### PR TITLE
Create edcp-students.php

### PR DESCRIPTION
**Pull request by cdavis-ubc**

### CODE REVIEW

File: templates/education/edcp-students.php [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+?>
+
+<div id="<?php echo absint( $post_id ); ?>" class="<?php echo esc_attr( $plain_term_slug ); ?> iso-profile">
+
+  <div class="boxey <?php echo esc_attr( $odd_even ); ?> <?php echo esc_attr( $plain_term_slug ); ?> profile">
+
+    <div class="boxey-inside">
+
+    	<small class="red-tags"><?php echo $plain_term_name ?></small>
```
Comment (richardtape) [inline templates/education/edcp-students.php]: Need escaping here

File: templates/education/edcp-students.php [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+
+    	 <div class="boxey-inner">
+            
+          <div class="iso-featured-image"><a href="#modal_<?php echo absint( $post_id ); ?>" class="fancybox-inline"><img src="<?php echo esc_url( $post_thumbnail ); ?>" alt="" /></a></div>
+          
+           <h3>
+
+            <div class="iso-title"><a href="#modal_<?php echo absint( $post_id ); ?>" class="fancybox-inline"><?php $the_title;?></a>
```
Comment (richardtape) [inline templates/education/edcp-students.php]: Need escaping for `$the_title` here

File: templates/education/edcp-students.php [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+           </div>
+
+           </h3>
+
+				<small><a href="mailto:<?php echo esc_attr( $email );?>"><?php echo esc_html( $email );?></a></small>
+
+				<div class="iso-description hidden"><?php $the_content;?></div>
```
Comment (richardtape) [inline templates/education/edcp-students.php]: `$the_content` doesn't exist in this template, does it?

File: templates/education/edcp-students.php [hunk 4]
```diff
@@ -0,0 +1,112 @@
...
+        
+           <div class="modal-body-content">
+          
+             <div class="span3"><img src="<?php echo esc_url( $post_thumbnail ); ?>" alt="" /></div>
+             
+             <div class="span9">
+                  
+                  <div class="modal-iso-description">
+
+                 <?php $the_content;?>
```
Comment (richardtape) [inline templates/education/edcp-students.php]: I can't see `$the_content` defined anywhere here

**Comment by cdavis-ubc** [comment]
escaping added to the_title and the_content

**Comment by richardtape** [comment]
Cool, let me know when you've also done it for $plain_term_name on line 
44 too and I'll merge

cdavis-ubc wrote:

**Comment by cdavis-ubc** [comment]
ok added esc to line 44 :)
thanks!

File: templates/education/edcp-students.php [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+
+<div style="display:none;">
+ 
+  <div id="modal_<?php echo absint( $post_id ); ?>" class="span9">
+ 
+    <div>
+
+    	<div class="modal-header">
+			
+			<h3 id="myModalLabel_<?php echo absint( $post_id ); ?>" class="modal-label header-tags">[the_title]
```
Comment (richardtape) [inline templates/education/edcp-students.php]: You have [the_title] as a shortcode here. That's not going to work ;)

**Comment by cdavis-ubc** [comment]
gaaah! that was dumb. ok just fixed it.

**Pull request merged**